src/poetry/console/commands/init.py

```diff
@@ -448,6 +453,15 @@ def _format_requirements(self, requirements: list[dict[str, str]]) -> Requiremen
 
         return requires
 
+    @staticmethod
+    def _validate_pep440_version(version: str) -> str:
+        try:
```

Comment:
this try-except (also the other) is pointless.  If all you do in the `except` branch is re-raise the exception, you might as well simply not catch it in the first place.

Reply:
I thought it would be better to clarify that the function to parse will generate an error if the format is incorrect.
Would it be better to simply call the function and write the intent in a comment?

Reply:
fixed.